airbyte-integrations/bases/base-normalization/integration_tests/resources/test_simple_streams/data_input/messages.txt

```diff
@@ -51,3 +51,4 @@
 {"type": "RECORD", "record": {"stream": "1_prefix_startwith_number", "emitted_at": 1602637990800, "data": { "id": 2, "date": "2020-09-01", "text": "hi 4"}}}
 
 {"type":"RECORD","record":{"stream":"multiple_column_names_conflicts","data":{"id":1,"User Id":"chris","user_id":42,"User id":300,"user id": 102,"UserId":101},"emitted_at":1623959926}}
+{"type":"RECORD","record":{"stream":"special_characters","data":{"id":1,"character_1":"some random special characters: ࠈൡሗ","user_id":42,"character_2":"some random special characters: žõ⥁🌞"},"emitted_at":1623959932}}
```

Comment:
If I'm not mistaken, this test will effect normalization of all destinations. Are you sure that we are safe to do it?

Reply:
Yes, you are right. @grubberr will check if it works for all destinations and will make changes in scope of the PR if not.